### PR TITLE
Increase movie update interval

### PR DESCRIPTION
**Pull request by Edholm**
No need to update movies that often. 12h is probably too often as well...

### CODE REVIEW

File: backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt [hunk 1]
```diff
@@ -63,29 +64,29 @@ class AsyncMovieUpdater(private val movieRepository: MovieRepository,
     }
   }
 
-  private fun isUpdateRequired(movie: Movie) = movie.needsMoreInfo() || movie.isMissingImdbId()
+  private fun isUpdateRequired(movie: Movie) = movie.needsMoreInfo() || (movie.isMissingImdbId() && movie.imdbId != "N/A")
```
Comment (Dr-Horv) [inline backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt]: Should perhaps use a constant for "N/A"?

File: backend/src/main/resources/application.properties [hunk 2]
```diff
@@ -14,3 +14,5 @@ login.defaultRedirectPath=${login_defaultRedirectPath:user}
 
 # Request API key here: https://www.themoviedb.org/documentation/api
 sefilm.tmdb.apikey=${tmdb_apikey}
+
+logging.level.didit.rocks=DEBUG
```
Comment (Dr-Horv) [inline backend/src/main/resources/application.properties]: Intentional?

Comment (Edholm) [inline backend/src/main/resources/application.properties]: semi-intentional... It did not have any effect. For some reason debug-lines for our own code isn't show, which is somewhat annoying.

File: backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt (hunk 1 above)
Comment (Edholm) [inline backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt]: yup, I just was a bit tired. I will move this to its own method I think.

Comment (lindskogen) [inline backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt]: Extract `"N/A"` to enum, and create helper method

File: backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt [hunk 3]
```diff
@@ -63,29 +64,29 @@ class AsyncMovieUpdater(private val movieRepository: MovieRepository,
...
 
   private fun updateInfo(movie: Movie) {
     if (movie.needsMoreInfo()) {
       fetchExtendedInfoForMovie(movie)
     }
-    if (movie.isMissingImdbId()) {
+    if (movie.isMissingImdbId() && movie.imdbId != "N/A") {
```
Comment (lindskogen) [inline backend/src/main/kotlin/rocks/didit/sefilm/schedulers/AsyncMovieUpdater.kt]: Extract `"N/A"` to enum, and create helper method

File: backend/src/main/kotlin/rocks/didit/sefilm/schedulers/ScheduledPopularityUpdater.kt [hunk 4]
```diff
@@ -56,12 +60,14 @@ class ScheduledPopularityUpdater(private val movieRepository: MovieRepository,
   private fun updatePopularity(movie: Movie) {
     val popularityAndId = when {
       movie.tmdbId != null -> fetchPopularityByTmdbId(movie)
-      !movie.isMissingImdbId() -> fetchPopularityByImdbId(movie)
+      !movie.isMissingImdbId() && movie.imdbId != "N/A" -> fetchPopularityByImdbId(movie)
```
Comment (lindskogen) [inline backend/src/main/kotlin/rocks/didit/sefilm/schedulers/ScheduledPopularityUpdater.kt]: Extract `"N/A"` to enum, and create helper method

File: backend/src/main/kotlin/rocks/didit/sefilm/schedulers/ScheduledPopularityUpdater.kt [hunk 5]
```diff
@@ -86,12 +92,17 @@ class ScheduledPopularityUpdater(private val movieRepository: MovieRepository,
   }
 
   private fun fetchPopularityByImdbId(movie: Movie): PopularityAndId? {
-    if (movie.imdbId == null) {
+    if (movie.isMissingImdbId() || movie.imdbId == "N/A" || movie.imdbId == null) {
```
Comment (lindskogen) [inline backend/src/main/kotlin/rocks/didit/sefilm/schedulers/ScheduledPopularityUpdater.kt]: Extract `"N/A"` to enum, and create helper method

**Comment by Edholm** [comment]
I forgot to verify that the IDs are serialised correctly. I will check that before merge I merge

**Pull request merged**